src/get-files-stream.js

```diff
@@ -0,0 +1,45 @@
+var File = require('vinyl')
+var vinylfs = require('vinyl-fs-that-respects-files')
+var vmps = require('vinyl-multipart-stream')
+var stream = require('stream')
+var Merge = require('merge-stream')
+
+exports = module.exports = getFilesStream
+
+function getFilesStream (files, opts) {
```

Comment:
@diasdavid did you change anything here, or can we just pull my stuff from #45 45?